bin/latex_to_unicode.py

```diff
@@ -177,6 +177,9 @@ def latex_to_unicode(s):
     # Transform errant citations into "(CITATION)"
     s = re.sub(r"\\(new)?cite.? ?\{([\w:-]+?)\}", r"(CITATION)", s)
```

Comment:
Because of line 153, the space after `\cite` is not optional.

Reply:
True, I added it for robustness against future changes.

Reply:
(Could be removed)